pkgs/development/libraries/zchunk/default.nix

```diff
@@ -20,6 +21,13 @@ stdenv.mkDerivation rec {
     hash = "sha256-7H1WF5VkpA65xCdEa0Sw4r4jj+kGhDVCMr5AeE+3Ii4=";
   };
 
+  # unbreak on darwin by finding argp-standaline
+  patches = lib.optional stdenv.isDarwin (fetchpatch {
+    name = "0001-meson-fix-argp-standalone-wrap-and-find_library.patch";
+    url = "https://github.com/buildroot/buildroot/raw/master/package/zchunk/0001-meson-fix-argp-standalone-wrap-and-find_library.patch";
```

Comment:
You're probably going to need to fix this URL to a specific SHA, otherwise if they ever update the patch this build will break?

Or download the patch and commt it to nixpkgs, so it doesn't need to grab it off the internet.

Reply:
Downloaded and committed it; there was some extraneous stuff in the patch that we don't need here anyway.